File: wordCounter/wordCounter.py
import string

if __name__ == "__main__":

	wordCounter = 0

	word = input("Enter word that you want to find: ")

	file = open("notepad.txt", "r")
	for line in file:
		if word in line: 
			wordCounter+=1
			print("The searched word '" + word + "' has been found " + str(wordCounter) + " times")

File: wordCounter/wordCounter.cpp
#include <iostream>
#include <fstream>
#include <string>

int main(){
	
	
	std::string line;
	std::string word;
	int wordStart;
	int wordCounter = 0;
	
	std::cout << "Enter word/phrase that you want to search for:\n";
	std::cin >> word;
	
	std::ifstream txtfile;
	txtfile.open("notepad.txt");
	if(txtfile.is_open())
	{
		while(!txtfile.eof())
		{
			getline(txtfile, line);
			if((wordStart = line.find(word, 0)) != std::string::npos)
			{
				wordCounter++;
				std::cout << "The word '" << word << "' has been found " << wordCounter << " times.\n";
			}
		}
		txtfile.close();
	}
	else
	{
		std::cout << "Could not open file.\n";
	}
	
	return 0;
}
